internal/kibana/synthetics/schema.go

```diff
@@ -176,6 +196,61 @@ func monitorConfigSchema() schema.Schema {
 	}
 }
 
+func browserMonitorFieldsSchema() schema.Attribute {
+	return schema.SingleNestedAttribute{
+		Optional:            true,
+		MarkdownDescription: "Browser Monitor specific fields",
+		Attributes: map[string]schema.Attribute{
+			"inline_script": schema.StringAttribute{
+				Optional:            false,
+				Required:            true,
+				MarkdownDescription: "The inline script.",
+			},
+			"screenshots": schema.StringAttribute{
+				Optional:            true,
+				MarkdownDescription: "Controls the behavior of the screenshots feature.",
+				Validators: []validator.String{
+					stringvalidator.OneOf("on", "off", "only-on-failure"),
+				},
+				Computed: true,
+				Default:  stringdefault.StaticString("on"),
```

Comment:
From the [docs](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework@v1.11.0/resource/schema#BoolAttribute):

> If the attribute is computed and the value could be altered by other changes then a default should be avoided and a plan modifier should be used instead.

The same goes for the other `Default` values in the schema. 

I'm not sure if these _need_ to be computed, but if they do we likely want something like:

```
PlanModifiers: []{stringplanmodifiers.UseStateForUnknown(), defaultModifier("on")}
```

Reply:
Thanks a lot, let me check. The issue I had - plan was different after creation, because API returns default value for this attribute, but I didn't set it.

Reply:
I"m sorry, it doesn't work. TF complains 
```
Schema Using
        Attribute Default For Non-Computed Attribute: Attribute "browser.screenshots"
        must be computed when using default.
```

Reply:
Ok, I'll leave only computed then, as a sign that API defines values, if not set.

Reply:
FYI: according to  https://github.com/hashicorp/terraform-plugin-framework/issues/285 there is no default for plan modifier and `UseStateForUnknown` should be enough. At least, I wasn't able to find default modifier in plugin framework. Please, let me know if I need to add it and how.

Reply:
@tobio , should be done, could you re-check?